frontend/src/containers/services/list.js

```diff
@@ -27,7 +30,11 @@ const Services = (props) => {
 
   return (
     <div>
-      {serviceList}
+      <ServicesView {...props}>
+        {serviceList}
+      </ServicesView>
+
```

Comment:
It seems that the <ServicesView> addition here duplicates the title and toggle on the list page